make/docker-compose.tpl

```diff
@@ -80,6 +80,8 @@ services:
       - /data/secretkey:/etc/ui/key:z
       - /data/ca_download/:/etc/ui/ca/:z
       - /data/psc/:/etc/ui/token/:z
+    extra_hosts:
+      - "PKS_UAA_HOST:PKS_UAA_IP_ADDR"
```

Comment:
better to name it 'UAA_HOSTNAME' and 'UAA_IP', because it might be PKS or PAS or other in future.

Reply:
but for PAS there is a DNS in the network?  So I believe this placeholder is for PKS only?

Reply:
just in case adding other UAA provider in future